EpiserverRedirects/Menu/RedirectRuleStore.cs

```diff
@@ -77,9 +78,17 @@ public ActionResult Put(RedirectRuleDto dto)
         [HttpDelete]
         public ActionResult Delete(Guid id)
         {
-            var deletedSuccessfully = id == _clearAllGuid
-                ? _redirectRuleRepository.ClearAll()
-                : _redirectRuleRepository.Delete(id);
+            bool deletedSuccessfully;
+            
+            if (id == _clearAllGuid) {
```

Comment:
I'm not fluent with EpiServer store concept but I wonder if this is possible to have separate methods in controller (I believe it needs to be supported by JS side of EpiServer store): Delete(GuidId), ClearAll() and RemoveAllDuplicates() ?